### PR TITLE
JBIDE-26667 Easyimport dependency fix

### PR DESCRIPTION
**Pull request by jkopriva**
Signed-off-by: Josef Kopriva <jkopriva@redhat.com>

# Pull Request Checklist

## Code
* Are method-/class-/variable-names meaningful?
* Are methods concise, not too long?
* Are catch blocks catching precise Exceptions only (no catch all)?
* Is copyright with correct year in new files?
* Is there a link to JIRA? (if needed)

## Testing
* Does it work?
* Are the tests green on Jenkins? Is there a link to Jenkins build?

### CODE REVIEW

**Comment by jkopriva** [comment]
Build https://dev-platform-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/Devstudio/view/devstudio_integration_tests/job/easyimport.itests/418/

**Comment by jkopriva** [comment]
@odockal can you please review?

File: tests/org.jboss.tools.easymport.ui.bot.test/pom.xml [hunk 1]
```diff
@@ -38,11 +38,6 @@
 							<artifactId>org.eclipse.jdt.feature.group</artifactId>
 							<version>0.0.0</version>
 						</dependency>
-						<dependency>
```
Comment (odockal) [inline tests/org.jboss.tools.easymport.ui.bot.test/pom.xml]: You do not need the dep anymore? Or it was useless for some time already?

Comment (jkopriva) [reply tests/org.jboss.tools.easymport.ui.bot.test/pom.xml]: @odockal this has moved to egit - I have seen some email about this.

**Pull request merged**